.moban.yaml

```diff
@@ -34,6 +34,7 @@ dependencies:
   - setuptools>=21.0.0
```

Comment:
first of all, separating your progress into multiple commits would be nicer to review, because I can then 'follow' your commits like a train of thought.

Now I basically have to understand everything you do at once.

I know that it's hard to get commits right when first writing new code, but if you haven't done so before, take a look at 'git rebase --interactive' to learn how to change previous commits in your branch.

Reply:
I will do that :+1:

Reply:
Done :+1: